### ISSUE TITLE
Translation Contracts (german)

### ISSUE THREAD

**Issue by ukkt** [glpi-project/glpi#10094]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

9.5.6

### Bug description

Hello,

I have some translation problems (german) for the contracts. The contracts are not translated (see pictures) and in the columns Name and Number the first 0 are cut.

In the version 9.5.4 I had no problems.
 

![pic1](https://user-images.githubusercontent.com/49140655/145776876-f0b8e452-0ead-4d65-9a85-2dd8fd5f6525.png)
![pic2](https://user-images.githubusercontent.com/49140655/145776878-9fc10ccd-82c1-4893-ac4d-2dcaed5fd3f0.png)

### Relevant log output

```shell
2021-12-13 09:01:12 [6@KABEL254]
includeLocales(applicationsbylocation_report_title, reports) => not found
2021-12-13 09:01:12 [6@KABEL254]
includeLocales(globalhisto_report_title, reports) => not found
2021-12-13 09:01:12 [6@KABEL254]
includeLocales(iteminstall_report_title, reports) => not found
2021-12-13 09:01:12 [6@KABEL254]
includeLocales(listequipmentbylocation_report_title, reports) => not found
2021-12-13 09:01:12 [6@KABEL254]
includeLocales(printers_report_title, reports) => not found
2021-12-13 09:01:12 [6@KABEL254]
includeLocales(searchinfocom_report_title, reports) => not found
2021-12-13 09:01:12 [6@KABEL254]
includeLocales(softnotinstalled_report_title, reports) => not found
2021-12-13 09:01:12 [6@KABEL254]
includeLocales(softversioninstallations_report_title, reports) => not found
2021-12-13 09:01:12 [6@KABEL254]
includeLocales(statnightticketsbypriority_report_title, reports) => not found
2021-12-13 09:01:12 [6@KABEL254]
includeLocales(statticketsbyentity_report_title, reports) => not found
2021-12-13 09:01:12 [6@KABEL254]
includeLocales(statticketsbypriority_report_title, reports) => not found
2021-12-13 09:01:12 [6@KABEL254]
includeLocales(transferreditems_report_title, reports) => not found
2021-12-13 09:01:12 [6@KABEL254]
includeLocales(zombies_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(applicationsbylocation_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(globalhisto_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(iteminstall_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(listequipmentbylocation_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(printers_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(searchinfocom_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(softnotinstalled_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(softversioninstallations_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(statnightticketsbypriority_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(statticketsbyentity_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(statticketsbypriority_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(transferreditems_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(zombies_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(applicationsbylocation_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(globalhisto_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(iteminstall_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(listequipmentbylocation_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(printers_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(searchinfocom_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(softnotinstalled_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(softversioninstallations_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(statnightticketsbypriority_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(statticketsbyentity_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(statticketsbypriority_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(transferreditems_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(zombies_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(applicationsbylocation_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(globalhisto_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(iteminstall_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(listequipmentbylocation_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(printers_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(searchinfocom_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(softnotinstalled_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(softversioninstallations_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(statnightticketsbypriority_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(statticketsbyentity_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(statticketsbypriority_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(transferreditems_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(zombies_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(applicationsbylocation_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(globalhisto_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(iteminstall_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(listequipmentbylocation_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(printers_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(searchinfocom_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(softnotinstalled_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(softversioninstallations_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(statnightticketsbypriority_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(statticketsbyentity_report_title, reports) => not found
2021-12-13 09:01:14 [6@KABEL254]
includeLocales(statticketsbypriority_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(transferreditems_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(zombies_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(applicationsbylocation_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(globalhisto_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(iteminstall_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(listequipmentbylocation_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(printers_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(searchinfocom_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(softnotinstalled_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(softversioninstallations_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(statnightticketsbypriority_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(statticketsbyentity_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(statticketsbypriority_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(transferreditems_report_title, reports) => not found
2021-12-13 09:01:15 [6@KABEL254]
includeLocales(zombies_report_title, reports) => not found
```

### Page URL

http://localhost/glpi/front/contract.php

### Steps To reproduce

- set the languarge to german
- go to contracts

### Your GLPI setup information

<html>
<body>

[code]   GLPI 9.5.6 (/glpi => C:\xampp\htdocs\glpi) Installation mode: TARBALL
--

Operating system: Windows NT KABEL254 10.0 build 14393 (Windows Server 2016) i586 PHP 7.4.26 apache2handler (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, apcu, bcmath, bz2, 	calendar, ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, imap, intl, json, ldap, libxml, 	mbstring, mysqli, mysqlnd, openssl, pcre, pdo_mysql, pdo_sqlite, readline, session, snmp, sodium, standard, tokenizer, xml, 	xmlreader, xmlrpc, xmlwriter, zip, zlib) Setup: max_execution_time="30" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files" 	upload_max_filesize="2M"  Software: Apache/2.4.33 (Win32) OpenSSL/1.1.1l PHP/7.4.26 (Apache/2.4.33 (Win32) OpenSSL/1.1.1l PHP/7.4.26 Server at kabel254 	Port 80) 	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.55 Safari/537.36 Edg/96.0.1054.43 Server Software: MySQL Enterprise Server - Commercial 	Server Version: 8.0.11-commercial 	Server SQL Mode:  	Parameters: root@kabel254/glpi 	Host info: kabel254 via TCP/IP 	 PHP version is at least 7.2.0 - Perfect! Sessions support is available - Perfect! Allocated memory > 64 Mio - Perfect! mysqli extension is installed ctype extension is installed fileinfo extension is installed json extension is installed mbstring extension is installed iconv extension is installed zlib extension is installed curl extension is installed gd extension is installed simplexml extension is installed intl extension is installed ldap extension is installed apcu extension is installed Zend OPcache extension is installed xmlrpc extension is installed exif extension is installed zip extension is installed bz2 extension is installed sodium extension is installed Database version seems correct (8.0.11) - Perfect!  The log file has been created successfully. Write access to C:\xampp\htdocs\glpi/files/_cache has been validated. Write access to C:\xampp\htdocs\glpi/config has been validated. Write access to C:\xampp\htdocs\glpi/files/_cron has been validated. Write access to C:\xampp\htdocs\glpi/files has been validated. Write access to C:\xampp\htdocs\glpi/files/_dumps has been validated. Write access to C:\xampp\htdocs\glpi/files/_graphs has been validated. Write access to C:\xampp\htdocs\glpi/files/_lock has been validated. Write access to C:\xampp\htdocs\glpi/files/_pictures has been validated. Write access to C:\xampp\htdocs\glpi/files/_plugins has been validated. Write access to C:\xampp\htdocs\glpi/files/_rss has been validated. Write access to C:\xampp\htdocs\glpi/files/_sessions has been validated. Write access to C:\xampp\htdocs\glpi/files/_tmp has been validated. Write access to C:\xampp\htdocs\glpi/files/_uploads has been validated. Write access to C:\xampp\htdocs\glpi/marketplace has been validated. Web access to files directory is protected

GLPI_ROOT: C:\xampp\htdocs\glpi GLPI_CONFIG_DIR: C:\xampp\htdocs\glpi/config GLPI_VAR_DIR: C:\xampp\htdocs\glpi/files GLPI_MARKETPLACE_DIR: C:\xampp\htdocs\glpi/marketplace GLPI_USE_CSRF_CHECK: 1 GLPI_CSRF_EXPIRES: 7200 GLPI_CSRF_MAX_TOKENS: 100 GLPI_USE_IDOR_CHECK: 1 GLPI_IDOR_EXPIRES: 7200 GLPI_ALLOW_IFRAME_IN_RICH_TEXT:  GLPI_TELEMETRY_URI: https://telemetry.glpi-project.org GLPI_INSTALL_MODE: TARBALL GLPI_NETWORK_MAIL: glpi@teclib.com GLPI_NETWORK_SERVICES: https://services.glpi-network.com GLPI_MARKETPLACE_PRERELEASES:  GLPI_MARKETPLACE_ALLOW_OVERRIDE: 1 GLPI_MARKETPLACE_MANUAL_DOWNLOADS: 1 GLPI_USER_AGENT_EXTRA_COMMENTS:  GLPI_AJAX_DASHBOARD: 1 GLPI_CALDAV_IMPORT_STATE: 0 GLPI_DEMO_MODE: 0 GLPI_FORCE_EMPTY_SQL_MODE: 1 GLPI_DOC_DIR: C:\xampp\htdocs\glpi/files GLPI_CACHE_DIR: C:\xampp\htdocs\glpi/files/_cache GLPI_CRON_DIR: C:\xampp\htdocs\glpi/files/_cron GLPI_DUMP_DIR: C:\xampp\htdocs\glpi/files/_dumps GLPI_GRAPH_DIR: C:\xampp\htdocs\glpi/files/_graphs GLPI_LOCAL_I18N_DIR: C:\xampp\htdocs\glpi/files/_locales GLPI_LOCK_DIR: C:\xampp\htdocs\glpi/files/_lock GLPI_LOG_DIR: C:\xampp\htdocs\glpi/files/_log GLPI_PICTURE_DIR: C:\xampp\htdocs\glpi/files/_pictures GLPI_PLUGIN_DOC_DIR: C:\xampp\htdocs\glpi/files/_plugins GLPI_RSS_DIR: C:\xampp\htdocs\glpi/files/_rss GLPI_SESSION_DIR: C:\xampp\htdocs\glpi/files/_sessions GLPI_TMP_DIR: C:\xampp\htdocs\glpi/files/_tmp GLPI_UPLOAD_DIR: C:\xampp\htdocs\glpi/files/_uploads GLPI_NETWORK_REGISTRATION_API_URL: https://services.glpi-network.com/api/registration/ GLPI_MARKETPLACE_PLUGINS_API_URI: https://services.glpi-network.com/api/glpi-plugins/ GLPI_I18N_DIR: C:\xampp\htdocs\glpi/locales GLPI_VERSION: 9.5.6 GLPI_SCHEMA_VERSION: 9.5.6 GLPI_MIN_PHP: 7.2.0 GLPI_YEAR: 2021

htmlawed/htmlawed version 1.2.5 in (C:\xampp\htdocs\glpi\vendor\htmlawed\htmlawed) phpmailer/phpmailer version 6.1.6 in (C:\xampp\htdocs\glpi\vendor\phpmailer\phpmailer\src) simplepie/simplepie version 1.5.6 in (C:\xampp\htdocs\glpi\vendor\simplepie\simplepie\library) tecnickcom/tcpdf version 6.3.5 in (C:\xampp\htdocs\glpi\vendor\tecnickcom\tcpdf) michelf/php-markdown in (C:\xampp\htdocs\glpi\vendor\michelf\php-markdown\Michelf) true/punycode in (C:\xampp\htdocs\glpi\vendor\true\punycode\src) iamcal/lib_autolink in (C:\xampp\htdocs\glpi\vendor\iamcal\lib_autolink) sabre/dav in (C:\xampp\htdocs\glpi\vendor\sabre\dav\lib\DAV) sabre/http in (C:\xampp\htdocs\glpi\vendor\sabre\http\lib) sabre/uri in (C:\xampp\htdocs\glpi\vendor\sabre\uri\lib) sabre/vobject in (C:\xampp\htdocs\glpi\vendor\sabre\vobject\lib) laminas/laminas-cache in (C:\xampp\htdocs\glpi\vendor\laminas\laminas-cache\src) laminas/laminas-i18n in (C:\xampp\htdocs\glpi\vendor\laminas\laminas-i18n\src) laminas/laminas-serializer in (C:\xampp\htdocs\glpi\vendor\laminas\laminas-serializer\src) monolog/monolog in (C:\xampp\htdocs\glpi\vendor\monolog\monolog\src\Monolog) sebastian/diff in (C:\xampp\htdocs\glpi\vendor\sebastian\diff\src) elvanto/litemoji in (C:\xampp\htdocs\glpi\vendor\elvanto\litemoji\src) symfony/console in (C:\xampp\htdocs\glpi\vendor\symfony\console) scssphp/scssphp in (C:\xampp\htdocs\glpi\vendor\scssphp\scssphp\src) laminas/laminas-mail in (C:\xampp\htdocs\glpi\vendor\laminas\laminas-mail\src\Protocol) laminas/laminas-mime in (C:\xampp\htdocs\glpi\vendor\laminas\laminas-mime\src) rlanvin/php-rrule in (C:\xampp\htdocs\glpi\vendor\rlanvin\php-rrule\src) blueimp/jquery-file-upload in (C:\xampp\htdocs\glpi\vendor\blueimp\jquery-file-upload\server\php) ramsey/uuid in (C:\xampp\htdocs\glpi\vendor\ramsey\uuid\src) psr/log in (C:\xampp\htdocs\glpi\vendor\psr\log\Psr\Log) psr/simple-cache in (C:\xampp\htdocs\glpi\vendor\psr\simple-cache\src) mexitek/phpcolors in (C:\xampp\htdocs\glpi\vendor\mexitek\phpcolors\src\Mexitek\PHPColors) guzzlehttp/guzzle in (C:\xampp\htdocs\glpi\vendor\guzzlehttp\guzzle\src) guzzlehttp/psr7 in (C:\xampp\htdocs\glpi\vendor\guzzlehttp\psr7\src) wapmorgan/unified-archive in (C:\xampp\htdocs\glpi\vendor\wapmorgan\unified-archive\src) paragonie/sodium_compat in (C:\xampp\htdocs\glpi\vendor\paragonie\sodium_compat\src)

Not active

Way of sending emails: SMTP (anonymous@srvxxxx)

 

reports              Name: Berichte                       Version: 1.13.1     State: Enabled 	order                Name: Bestellverwaltung              Version: 2.6.0      State: Installed / not activated 	datainjection        Name: Data injection                 Version: 2.9.0      State: Installed / not activated 	formcreator          Name: Form Creator                   Version: 2.10.1     State: Installed / not activated 	accounts             Name: Konten                         Version: 2.6.0      State: Enabled 	archires             Name: Network Architectures          Version: 2.7.0      State: Enabled 	protocolsmanager     Name: Protocols manager              Version: 1.4        State: Enabled
[/code]

</body>
</html>

### Anything else?

no

**Comment by ukkt** [comment]
The translation problem also exists in GLPI 10.

**Comment by cconard96** [comment]
Number display issue should be fixed by #9570 which will be included in GLPI 10 and the next 9.5.X version.

I checked Transifex (the platform GLPI uses for localizations) and the text "Name" doesn't have a translation yet. "Contract" does have a translation now.

If you wish to contribute translations, you can from [Transifex](https://www.transifex.com/glpi/GLPI/) by requesting access to translate for German.
Direct link: https://www.transifex.com/glpi/GLPI/language/de_DE/
To request access, click the 3 dots menu next to "View Members" and select "Join team".
